blog/zh/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md

```diff
@@ -14,9 +14,10 @@ keywords:
 - ARM
 - Azure
 - AWS
+- Oracle
 - Google
 - Apache APISIX
-description: 本文使用 API 网关 Apache APISIX 来比较 GCP、AWS 和 Azure ARM 架构服务器在网络 IO 密集型场景下的性能。
+description: 本文使用 API 网关 Apache APISIX 来比较 GCP、AWS、Azure 和 OCI ARM 架构服务器在网络 IO 密集型场景下的性能。
```

Comment:
Need to change the title too
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/39793568/196367534-3093495c-281e-44ac-b424-88ec50b7d29d.png">